x-pack/docs/en/rest-api/security/get-builtin-privileges.asciidoc

```diff
@@ -120,6 +120,7 @@ A successful call returns an object with "cluster" and "index" fields.
     "index",
     "maintenance",
     "manage",
+    "manage_dlm",
```

Comment:
Oh ya good q! I took another look: I think we should remove it from the list above **and** also remove the `manage_dlm` docs portion from `privileges.asciidoc` (I'll add a separate suggestion in a comment to remove it).

The reason is that docs don't work well with feature flags when it comes to pages that are themselves not feature flagged but have sections that need to be feature flagged. AFAIK there is no way to conditionally skip skip a section of a page, so I think it's best to omit the section from docs in such cases. Otherwise, we will end up with docs for a privilege that isn't actually available to end-users yet.

I'll create a separate Jira issue to complete docs for DLM privileges, so we don't lose track of the docs gap as we get closer to removing the feature flag.

Note that the DLM-specific pages are fine, and we can reference `manage_dlm` there already. Those are marked experimental and will therefore not show up in our "real" docs AFAIK.

Reply:
I created #96386 so that we don't forget to change this in the future.

Reply:
Hm, I'm curious if the tests will pass if we don't mention it - note that the feature flag is always enabled for SNAPSHOT builds

Reply:
The doc test is currently disabled due to other feature flagged privileges